matter_server/server/device_controller.py

```diff
@@ -46,7 +61,8 @@ def __init__(
         """Initialize the device controller."""
         self.server = server
         # Instantiate the underlying ChipDeviceController instance on the Fabric
-        self.chip_controller = server.stack.fabric_admin.NewController()
+        assert PAA_ROOT_CERTS_DIR.is_dir()
```

Comment:
Maybe raise `RuntimeError` if it's missing instead? Asserts can be turned off.

Reply:
This will do I/O, btw, so we should schedule it on the executor.

Reply:
This is in the init where we do a lot more IO before we start all async stuff

Reply:
Right, I see it's before we start the server. 👍